### ISSUE TITLE
multiple then not fulfilled

### ISSUE THREAD

**Issue by laoshu133** [vuejs/vue-resource#248]
```
var promise = Vue.Promise.resolve(1);
var x1;

promise.then(function (x) {
    x1 = x;
});

promise.then(function (x2) {
    console.log(x1 === x2, x1, x2);  // x2 is undefined
});
```

**Comment by Lidbetter** [comment]
Same issue.

Subsequent then's are called, however the response is undefined for all but the first then.

**Comment by maple-leaf** [comment]
confirmed here.

**Comment by steffans** [comment]
This is implemented intentionally, currently the promise chain does not break. Means your example code will actually work like this. Since the first `then` does not return anything.

```js
var promise = Vue.Promise.resolve(1);
var x1;

promise.then(function (x) {
    x1 = x;
}).then(function (x2) {
    console.log(x1 === x2, x1, x2);  // x2 is undefined
});
```

**Comment by laoshu133** [comment]
This is useful, but sometimes I need to do some caching;
And this implemented does not follow "Promise A+".

**Comment by steffans** [comment]
I removed this in the develop branch, will be released with v0.9
